### PR TITLE
Pricing changes

### LINKED ISSUE #1171: Update pricing
Post updated pricing docs before release on September 1st. 
- Users will no longer be charged for query duration. 
- Users will be charged for the following:
   - Total number of query requests (including tasks)
   - Total amount of data transferred out
- Request from Tim:
   - Contact Balaji for pricing deck walkthrough
   - Work with marketing to ensure language aligns. 

Notes:
- CSV Download does not affect charges. (Initial query brings the data to the browser. The CSV Download button simply packages up the data resident in the browser with no additional Flux query invocation.) Only charge for the initial query (API call - 1 count) and the data transfer cost.
- Writing Data Back into Influx does not affect charges. Only charge for initial query (API call - 1 count). No data transfer cost bc data is not transferred (moved through an internal gateway call).

##### Relevant URLs
We could add details on this page (we currently don't tell them how they're charged).
https://v2.docs.influxdata.com/v2.0/account-management/pricing-plans/

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #1171

### CODE REVIEW

File: content/v2.0/account-management/data-usage.md [hunk 1]
```diff
@@ -18,15 +18,13 @@ right corner of your {{< cloud-name "short" >}} user interface (UI) and select *
 
 Find data usage information for the time frame specified in the drop-down at the top of the Usage page.
 
-- **Writes:** Total data in MB written to your {{< cloud-name "short" >}} instance.
-- **Reads:** Total data in MB sent as responses to queries from your {{< cloud-name "short" >}} instance.
-- **Query Duration:** Total time spent processing queries in seconds.
+- **Data In:** Total data in MB written to your {{< cloud-name "short" >}} instance.
+- **Data Out:** Total data in MB sent as responses to queries from your {{< cloud-name "short" >}} instance.
+- **Query Count:** Total number of events.
```
Comment (sanderson) [inline content/v2.0/account-management/data-usage.md:23]: "Events" seems really general. Is this the description they have on the new pricing page? What about "Total number of query executions?"

File: content/v2.0/account-management/pricing-plans.md [hunk 2]
```diff
@@ -46,15 +46,24 @@ Data retention is determined by the time at which data is written to InfluxDB; n
...
 
-#### Usage-based Plan rate limits
+### Pricing vectors
 
-To protect against any intentional or unintentional harm, Usage-based Plans include soft rate limits:
+The following two vectors determine pricing:
 
-- **Writes:** 300MB every 5 minutes
+- **Data out** is the total sum of the data (measured in GB) returned to the user to answer a query, also known as data transfer costs.
+- **Query count** is the total number of individual query operations. Each individual operation — including
```
Comment (sanderson) [inline content/v2.0/account-management/pricing-plans.md:58]: I suggested "execution" above, but I'm not partial to "operation" or "execution" as long as we're consistent.

File: content/v2.0/account-management/pricing-plans.md [hunk 3]
```diff
@@ -46,15 +46,24 @@ Data retention is determined by the time at which data is written to InfluxDB; n
...
+queries, tasks, alert checks, notifications, and Data Explorer activity — is one billable query operation. Refreshing a dashboard with multiple cells counts as multiple query operations. Failed query or task executions do not add to your query count.
```
Comment (sanderson) [inline content/v2.0/account-management/pricing-plans.md:59]: Should we explicitly state that each dashboard cell will count as 1 query execution/operation on refresh?

File: content/v2.0/account-management/data-usage.md (hunk 1 above)
Comment (gunnaraasen) [reply content/v2.0/account-management/data-usage.md:23]: I agree there could be more detail on this. Maybe summarize this description?

> Query Count is billed on the total number of individual query operations, which include queries, tasks (alerts, notifications) and Data Explorer activity. Any individual operation, such as the execution of an alert task, will be counted as one billable query operation. Refreshing a dashboard with multiple cells will incur multiple query operations. Any failed query/task executions are not counted.

File: content/v2.0/account-management/pricing-plans.md (hunk 2 above)
Comment (gunnaraasen) [reply content/v2.0/account-management/pricing-plans.md:58]: We should also note that a query run through an API request counts as an operation. Though a single request can contain multiple flux queries.

**Pull request merged**